helpers/config.go

```diff
@@ -29,6 +29,7 @@ type ConfigDCP struct {
 	Group                  ConfigDCPGroup    `yaml:"group"`
 	BufferSizeKb           int               `yaml:"bufferSizeKb" default:"16384"`
 	ConnectionBufferSizeKb uint              `yaml:"connectionBufferSizeKb" default:"20480"`
+	ConnectionTimeout      int64             `yaml:"ConnectionTimeout" default:"5"`
```

Comment:
`yaml:"ConnectionTimeout" default:"5"` should be `yaml:"connectionTimeout" default:"5"`

Reply:
and we can make it time.Duration not int64